app/pages/project/project-page.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import { Markdown } from 'markdownz';
+import { sugarClient } from 'panoptes-client/lib/sugar';
+import ProjectNavbar from './project-navbar';
+import PotentialFieldGuide from './potential-field-guide';
+
+export default class ProjectPage extends React.Component {
+  constructor() {
+    super();
+    const lastSugarSubscribedID = null;
```

Comment:
Variable is declared just for this method but never used. Is it meant to be an object property instead?

Reply:
Ah, thanks for picking this up. I wasn't sure how to convert [this bit](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/project-page.cjsx#L56). Shall I just declare it inside [`updateSugarSubscription`](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/project-page.cjsx#L58)?

Reply:
`this.lastSugarSubscribedID = null;` Also, convention is to underscore private properties, since JavaScript doesn't have private properties and methods, so it looks like this should be named `this._lastSugarSubscribedID`.

Reply:
OK, doesn't underscore trigger a linter warning for you? I removed the `_` because of that.

Reply:
I ignore those linter warnings.